materialize-slack/Dockerfile

```diff
@@ -0,0 +1,39 @@
+ARG BASE_IMAGE=ghcr.io/estuary/base-image:v1
+
+# Build Stage
+################################################################################
+FROM golang:1.17-bullseye as builder
```

Comment:
This dockerfile will need updated to be like the current ones on the main branch, which use go 1.20, and the directory structure of the imports ("Build the connector projects we depend on.") has changed. See https://github.com/estuary/connectors/blob/60728e31593cc7c9a3d78c961b47eab3e84cca07/materialize-pinecone/Dockerfile#L1-L16.

Reply:
I think I had already updated this. The change you're talking about is moving things from e.g `go-materialize-boilerplate` to `go/materialize-boilerplate` right?